examples/chart-helm/README.md

```diff
@@ -0,0 +1,83 @@
+# chart-vllm
```

Comment:
It would be great to move this under `docs/` so it can be part of https://docs.vllm.ai

Reply:
I just moved the document under docs folder with a related rst file.